### PR TITLE
add config example

### PR DESCRIPTION
**Pull request by james03160927**
## Description

fixes https://github.com/gruntwork-io/cloud-nuke/issues/786.

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

File: config/examples/simple.yaml
Comment (stefan-nordcloud) [inline config/examples/simple.yaml:5]: According to [the README](https://github.com/gruntwork-io/cloud-nuke/blob/master/README.md?plain=1#L638), the `s3` names_regex will filter on the bucket name, and not the object key. Are you sure this will work?

[This part of the README](https://github.com/gruntwork-io/cloud-nuke/blob/master/README.md?plain=1#L500) also says
> The intention is to delete all the **s3 buckets** that match the include rules but not the exclude rules

File: aws/resource_registry.go
Comment (wakeful) [inline aws/resource_registry.go:67]: can you rebase? both **Grafana** and **ElasticCacheServerless** where already merge

File: config/examples/simple.yaml
Comment (stefan-nordcloud) [reply config/examples/simple.yaml:5]: I don't know what happened, but it looks like we had some regression here 🤷

**Review by denis256 — APPROVED** [review]

**Review by stefan-nordcloud — CHANGES_REQUESTED** [review]
Hi all, I just saw this was approved now. Please note my comments [here](https://github.com/gruntwork-io/cloud-nuke/pull/788/files#r1849005226) and [here](https://github.com/gruntwork-io/cloud-nuke/pull/788/files#r1849000844).

I don't know what happened. They were initially fixed when I made these comments and the conversation resolved, but now it seems like those commits have disappeared. Could you take a look at that before merging, please? Thanks!

**Comment by james03160927** [comment]
Hey @denis256, can you please approve the PR once again?

**Review by denis256 — APPROVED** [review]

**Review by stefan-nordcloud — APPROVED** [review]
For what its worth, I approve too 😉

**Pull request merged**
